web_console/api/job.js

```diff
@@ -23,12 +24,17 @@ router.get('/api/v1/jobs', SessionMiddleware, async (ctx) => {
   const jobs = await Job.findAll({
     order: [['created_at', 'DESC']],
   });
-  const { flapps } = await k8s.getFLAppsByNamespace(namespace);
-  const data = jobs.map((job) => ({
-    ...(flapps.items.find((item) => item.metadata.name === job.name)),
-    localdata: job,
-  }));
-  ctx.body = { data };
+  try {
+    const { flapps } = await k8s.getFLAppsByNamespace(namespace);
+    const data = jobs.map((job) => ({
+      ...(flapps.items.find((item) => item.metadata.name === job.name)),
+      localdata: job,
+    }));
+    ctx.body = { data };
+  } catch (e) {
+    ctx.status = 400;
+    ctx.body = await parseStream(e.response.body);
```

Comment:
what if using ky's [afterResponse](https://github.com/sindresorhus/ky#hooksafterresponse) hook? 'caz it's a common error response of k8s api server

Reply:
done

Reply:
> what if using ky's [afterResponse](https://github.com/sindresorhus/ky#hooksafterresponse) hook? 'caz it's a common error response of k8s api server

done